test-requirements.txt

```diff
@@ -1 +1,2 @@
 odoo_test_helper
+odoo-addon-partner_contact_personal_information_page @ git+https://github.com/OCA/partner-contact.git@refs/pull/1900/head#subdirectory=partner_contact_personal_information_page
```

Comment:
I'm curious about this directive. from what I understand it allows the test to be green by taking into account an unmerged dependency. Do you remove this commit before merging or the bot take care of it ?

Reply:
@houssine78 The process is described here: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

Indeed, this is used in order to:

- show tests are passing
- be able to test the PR functionally with runboat.